content/en/real_user_monitoring/browser/setup/server.md

```diff
@@ -114,20 +182,88 @@ To update your RUM Application:
 {{% /tab %}}
 {{< /tabs >}}
 
-## Limitations
+## Troubleshooting
 
-The available functionality has the following important limitations:
+### Nginx stops responding
 
-- If serving compressed traffic, the Auto-Instrumentation method is not able to inject the JS scriptlet into the HTML traffic.
-- This instrumentation method does not support any [advanced RUM configurations][3]. However, `allowedTracingUrls` and `excludedActivityUrls` are supported for Nginx web servers.
-- Auto-Instrumentation does not inject into encrypted requests served by Nginx or IIS related to TLS.
-- (Nginx only) Auto-Instrumentation does not inject encrypted requests served by the Nginx web server.
-- (Windows IIS only) Configuration for Auto-Instrumentation is only available per Windows IIS site.
+Since the module is in Preview, it's possible nginx may stop serving requests, particularly after installation. If you experience this issue, contact [Datadog support][4] with the following information to help us investigate and resolve the issue:
```

Comment:
```suggestion
Since the module is in Preview, it's possible NGINX may stop serving requests, particularly after installation. If you experience this issue, contact [Datadog support][4] with the following information to help us investigate and resolve the issue:
```